pg_diffix/query/anonymization.h

```diff
@@ -18,6 +18,11 @@ typedef struct AnonQueryLinks AnonQueryLinks;
  */
 extern AnonQueryLinks *compile_query(Query *query, List *personal_relations);
 
+/*
+ * Calls `rewrite_plan` for each item in a list of Plan nodes.
+ */
+extern void rewrite_plan_list(List *plans, AnonQueryLinks *links);
+
 /*
  * Wraps anonymizing Agg nodes with BucketScan nodes. Does nothing if links is NULL.
  * Frees memory associated with links after rewriting.
```

Comment:
> Frees memory associated with links after rewriting.

Not sure if the comment is out of date or if it really does it. In that case we can't call it twice.

Reply:
I don't see it doing any such thing, meaning the function doc is wrong.